internal/cli/server/service.go

```diff
@@ -23,6 +24,8 @@ import (
 
 const chunkSize = 1024 * 1024 * 1024
 
+var ErrUnavailable = errors.New("the service is currently unavailable, try again later")
```

Comment:
Let's be more specific on this error, e.g.

```
var ErrBorServiceUnavailable = errors.New("Bor service is currently unavailable, try again later")
```

Reply:
Done, thanks!